sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/datasketchesAggregates.scala

```diff
@@ -189,21 +189,20 @@ object HllSketchAgg {
   private val minLgConfigK = 4
   private val maxLgConfigK = 21
 
-  // Replicate Datasketche's HllUtil's checkLgK implementation, as we can't reference it directly
+  // Replicate Datasketches' HllUtil's checkLgK implementation, as we can't reference it directly.
   def checkLgK(lgConfigK: Int): Unit = {
     if (lgConfigK < minLgConfigK || lgConfigK > maxLgConfigK) {
-      throw new SketchesArgumentException(
-        s"Log K must be between $minLgConfigK and $maxLgConfigK, inclusive: " + lgConfigK)
+      throw QueryExecutionErrors.hllInvalidLgK(function = "HLL_SKETCH_AGG",
```

Comment:
```suggestion
      throw QueryExecutionErrors.hllInvalidLgK(function = "hll_sketch_agg",
```
I missed this before (see also my other comment): we want to use the pretty name here, which is all lowercase.

Reply:
Done.